app/Models/Screening.php

```diff
@@ -30,6 +30,7 @@ class Screening extends Model
         'serial_id',
         'image_id',
         'preScreeningOf',
+        'rating',
```

Comment:
Ich finde "rating" zu schwammig. Ich würds "fsk" oder "fskRating" nennen. Wie man unten in der React-Komponente sieht, hattest du das ja auch schon überlegt ;)